app.js

```diff
@@ -156,10 +159,12 @@ app.route("/edit")
                 });
             }
         }
-        var banner = await getTag(req.user.id).catch(function () {
-            res.status(404).render("notfound.pug");
-            return
-        });
+        setTimeout(() => {
+            getTag(req.user.id).catch(function () {
+                res.status(404).render("notfound.pug");
+                return
+            })
+        }, 2000);
```

Comment:
Adds a 2 second (2000 Millisecond) pause before the tag generator runs. This allows the request to the CMOC host server to complete and for the mii to be rendered before the tag is regenerated.
This shouldn't create any issues for other users (and I didn't see any in testing with multiple tabs) as setTimeout should let the code run the redirect before the timer finishes meaning there is no visible pause. res.redirect would need a timeout to prevent the need to refresh to see the new tag, but that shouldn't be an issue considering that was already a thing before commit 8b65bac.